tests/kuttl/common/assert-sample-deployment.yaml

```diff
@@ -215,3 +205,31 @@ status:
       public: http://ironic-inspector-public-openstack.apps-crc.testing
   databaseHostname: openstack
   readyCount: 1
+---
+apiVersion: kuttl.dev/v1beta1
+kind: TestAssert
+namespaced: true
+commands:
+  - script: |
+      set -x
+      RETURN_CODE=0
+      PUBLIC_URL=$(oc get -n openstack ironics.ironic.openstack.org ironic -o json | jq -r .status.apiEndpoints.ironic.public)
```

Comment:
oc has in-built jsonpath support, so you don't need to pipe to jq, you can:

`oc get -n openstack ironics.ironic.openstack.org ironic -o jsonpath='{.status.apiEndpoints.ironic.public}'`